src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -66,6 +90,9 @@ class RA_GUIBASE_API KeyMappingManager
     /// Add a callback, triggered when configuration is load or reloaded.
     void addListener( Listener callback );
 
+    /// Remove a callback, triggered when configuration is load or reloaded.
```

Comment:
tu be update wrt new observable, or removed for now

Reply:
Which _new observable_ ? Nothing changes about that into master ... Or did I miss something ?

Reply:
Well, there is no m_listeners in KeyMappingManager anymore ...

Reply:
? Not in master. cf line 149 of Radium-Engine/src/GuiBase/Utils/KeyMappingManager.hpp